jwst/tweakreg/tweakreg_step.py

```diff
@@ -31,9 +27,11 @@ def _oxford_or_str_join(str_list):
 
 
 SINGLE_GROUP_REFCAT = ["GAIADR3", "GAIADR2", "GAIADR1"]
+"""Astrometric catalog names"""
+
 _SINGLE_GROUP_REFCAT_STR = _oxford_or_str_join(SINGLE_GROUP_REFCAT)
 
-__all__ = ["TweakRegStep"]
+__all__ = ["TweakRegStep", "SINGLE_GROUP_REFCAT"]
```

Comment:
Why is this being added to the `__all__`?  It seems to me that this should be documented in the docstring of the `spec` attribute `abs_refcat` (which currently it already is, and that list is up-to-date).

Reply:
Oh ok, thanks, if it's already mentioned in those docs then I'm fine with it being in the all.  I wonder if (not as part of this PR) we should consider renaming it to something more descriptive, like "KNOWN_CATALOGS" or idk